api/lib/domain/usecases/update-organization-learners-password.js

```diff
@@ -0,0 +1,79 @@
+import { UserNotAuthorizedToUpdatePasswordError } from '../errors.js';
+import {
+  ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
+  ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE,
+  USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
```

Comment:
Mettre un ordre plus logique pour les codes d'erreur en mettant l'ordre dans lequel les erreurs peuvent arriver : 

```suggestion
  USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
  ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
  ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE,
```